transforms/__testfixtures__/class/class-pure-mixin3.output.js

```diff
@@ -0,0 +1,20 @@
+// for this file we disable the `pure-component` option
+// so we should not convert to a plain class
+var ReactCreateClass = require('react-create-class');
+var ReactComponentWithPureRenderMixin = require('ReactComponentWithPureRenderMixin');
+
+var ComponentWithOnlyPureRenderMixin = ReactCreateClass({
+  mixins: [ReactComponentWithPureRenderMixin],
+
+  getInitialState: function() {
+    return {
+      counter: this.props.initialNumber + 1,
+    };
+  },
+
+  render: function() {
+    return (
+      <div>{this.state.counter}</div>
```

Comment:
If the file uses JSX then we shouldn't prune `React` since we'll need `React.createElement` in scope